lib/karaokium_web/views/layout_view.ex

```diff
@@ -7,13 +7,13 @@ defmodule KaraokiumWeb.LayoutView do
 
   def menu(conn) when is_logged_in(conn) do
     base_menu(conn) ++
-      Enum.reduce(Enum.sort(conn.assigns.current_user.permissions), [], fn role, acc ->
+      Enum.reduce(Enum.sort(conn.assigns.current_user.permissions), [], fn role, _ ->
         case role do
           :admin ->
-            acc ++ admin_menu(conn)
+            admin_menu(conn)
 
           :sysadmin ->
-            acc ++ sysadmin_menu(conn)
```

Comment:
You shouldn't change this. Each permission opens the possibility to navigate to different places and do different things.

Reply:
Ok. I am commiting it to another branch since this branch is for register_page and not for navbar.